src/bridge/MatrixHandler.ts

```diff
@@ -1031,7 +1031,8 @@ export class MatrixHandler {
 
         // Generate an array of individual messages that would be sent
         const potentialMessages = ircClient.getSplitMessages(ircRoom.channel, ircAction.text);
-        const lineLimit = ircRoom.server.getLineLimit();
+        const roomLineLimit = await this.ircBridge.roomConfigs.getLineLimit(event.room_id, ircRoom);
```

Comment:
I'm a little worried about the delay in handling messages if room state is too slow, but hopefully the query will be subsecond quick.

Reply:
I wonder if this just needs a timeout.

Reply:
I added a timeout.